src/subs/FitSubroutines.cpp

```diff
@@ -2757,6 +2779,14 @@ Astro::StarCatalog Astro::getStarCatalog(const astrometry::MCat &matches,
         }
         starDOF.push_back(dof);
         starChisq.push_back(chisqThis);
+
+        double epoch;
+        for (auto const &detptr : *m) {
+            epoch = extensionEpochs[detptr->catalogNumber];
+            if (epoch) break;
```

Comment:
There's no chance of having more than one epoch you care about in this loop, right?

Reply:
Right.